docs/conf.py

```diff
@@ -9,12 +9,11 @@
 # If extensions (or modules to document with autodoc) are in another directory,
```

Comment:
Please remove the helper comments. You can keep the title e.g. 'Project Information' or 'General Configuration', but remove the descriptions. If required, you can add a link to the original conf file with the details documentation at the top of the file.